storage/tree_validation_test.go

```diff
@@ -26,29 +26,12 @@ import (
 	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/anypb"
 	"google.golang.org/protobuf/types/known/durationpb"
-	"google.golang.org/protobuf/types/known/emptypb"
 	"google.golang.org/protobuf/types/known/timestamppb"
 
-	ktestonly "github.com/google/trillian/crypto/keys/testonly"
-
 	_ "github.com/google/trillian/crypto/keys/der/proto"
 	_ "github.com/google/trillian/crypto/keys/pem/proto"
 )
 
-const (
-	privateKeyPath = "../testdata/log-rpc-server.privkey.pem"
```

Comment:
Is this file still used? Remove it too?

Reply:
Yes, still used in some `crypto/keys/pem` tests.